lib/gpins_control_device.h

```diff
@@ -58,6 +58,8 @@ class GpinsControlDevice : public thinkit::ControlDevice {
   absl::Status SendPacket(absl::string_view interface,
                           absl::string_view packet) override;
 
+  bool SupportsSendPacket() const override { return true; }
```

Comment:
All gpins should be renamed to pins including the file name. I'll approve and merge this PR but let's followup with a new PR for this change.